Scheduler/feature_scheduler/auxtel/fbs_config_image_photocal_survey.py

```diff
@@ -166,6 +166,19 @@ def get_scheduler():
             nexp=1,
             moon_distance=20.0,
         ),
+        Target(
+            target_name="PN Hen 2-113",
+            survey_name="BLOCK-273",
+            ra=Angle("14:59:53.4", unit=units.hourangle),
+            dec=Angle("-54:18:07", unit=units.deg),
+            hour_angle_limit=spec_ha_limit,
```

Comment:
Are you sure 6 hours of hour angle is enough to get this target now? It might be, I don't know, just asking to make sure.

Reply:
Yes, it should be sufficient, I just checked.